docs/movement-node/run/manul/suzuka-full-node.service.template

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
It would be nice to have a single template.

Reply:
> It would be nice to have a single template.

I could use envsubst to generate the final service file for anisble version also. 
The other way around would add ansible as another prerequisite for running the manual step and it would not make sense. 
Please let me know if you want me to make this change